index.rst

```diff
@@ -48,11 +48,302 @@
+  - Azimuth dependency. The corresponding force component are ``evt_azimuthForces.axial`` and ``evt_azimuthForces.tangent``. (To be implemented)
+  - Temperature dependency. The corresponding force component are ``evt_thermalForces.axial`` and ``evt_thermalForces.tangent``.
+  - Static bending forces. The corresponding force component is ``evt_staticForces.axial`` and ``evt_staticForces.tangent``. In Harris' lauguage, this includes `F0 <https://github.com/lsst-ts/ts_mtm2_cell/blob/master/configuration/lsst-m2/config/parameter_files/luts/FinalOpticalLUTs/F_0.csv/>`__ and `FF <https://github.com/lsst-ts/ts_mtm2_cell/blob/master/configuration/lsst-m2/config/parameter_files/luts/FinalOpticalLUTs/F_F.csv/>`__.
+  - Load cell zero offset. The corresponding force component is ``evt_loadCellZeroOffset``. To be implemented. See `Sec. 10 of SITCOMTN-004 <https://sitcomtn-004.lsst.io/#load-cell-zero-offsets/>`__.
+
+The command ``cmd_applyForces`` is not only intended for closed-loop AOS. It is also supposed to be used for manual AOS test, and bump test, where an arbituary set of forces need to be applied.
+
+There are no forces for slewing, because M2 slews passively. The actuators are much stiffer than M1M3. When the TMA slews, the M2 hard points will sense the load and the FB system will offload the forces and moments.
+
+The forces, as measured by the load cells, are given as telemetry under ``tel_forceActuatorData.axial`` and ``tel_forceActuatorData.tangent``.
+
+The M2 LUT forces are on whenever the mirror control system is in the enabled state.
+By default, the elevation angle used by the elevation dependent component of the LUT is from the onboard inclinometer.
+Only when the elevation angle from the onboard inclinometer is not available would the TMA telemetry be used.
+The source for the elevation angle can be switched between the onboard inclinometer and the TMA telemetry using the
+command ``cmd_selectElevationSource``, which takes one input parameter - 1=Onboard, 2=MTMount.
+The change in the telemetry source triggers the event ``evt_elevationSource``.
+
+######################
+M2 and Camera Hexapods
+######################
+
+The M2 and Camera hexapods have identical control interfaces. The Camera hexapod has ID=1 and M2 hexapod has ID=2.
+
+What the hexapods do is to position themselves to help achieve optimal image quality. So our discussion focuses on how we command the hexapods to the desired positions.
+Here and in the rest of this tech note, position of the hexapod refers to the position (x,y,z) and orientation (dx, dy, dz) relative to the pivot point.
```

Comment:
Re: rx, ry, rz 
Thanks that is a good catch. I was thinking about rx,ry,rz the whole time but somehow wrote dx,dy,dz.
I understand your reasons for using u,v,w. However, I am going to stick with rx,ry,rz for now, since that is what we have been using in AOS, and was the concensus between the authors of sitcomtn-004. See https://sitcomtn-004.lsst.io/#units-dof-names-and-cs

Re: relative to the pivot point
I think it would be clear if I say "rotation around the pivot point".
"orientation of the pivot point" sounds like the pivot point, as a point, has orientation.

Reply:
True. Certainly the move command specified the position and orientation of the coordinate system tied to the movable part of the hexapod and centered at the pivot point. I can see why you want to discuss that coordinate system.